packages/core/config/multisite/getEnv.test.js

```diff
@@ -0,0 +1,22 @@
+import getEnv from './getEnv';
+
+describe('getEnv', () => {
+  it('merge site-specific keys into process.env and return the resulting object', () => {
+    const mergedEnv = getEnv('multisite-one.irving.test');
+    expect(mergedEnv).toMatchObject({
+      API_ROOT_URL: 'https://irving-multisite.test/api',
+      ROOT_URL: 'https://multisite-one.irving.test',
+      BASIC_AUTH_USERNAME: 'alley',
+      BASIC_AUTH_PASSWORD: 'interactive',
+      BASIC_AUTH: true,
```

Comment:
Since we actually use this as basic auth for some of our staging environments, it shouldn't be in source code in a public repo.

Reply:
@kevinfodness ah the username/pass? absolutely, I'll change those to something else.